src/Workspaces/Core/Portable/Log/FunctionId.cs

```diff
@@ -312,5 +312,8 @@ internal enum FunctionId
         NonFatalWatson,
         GlobalOperationRegistration,
         CommandHandler_FindAllReference,
+
+        Infobar_EnableCodefix,
+        Infobar_EnableCodefixAndIgnoreFutureErrors,
```

Comment:
We may use the InfoBar for more stuff someday, so can we swap to:

```CodefixInfobar_Enable```
```CodefixInfobar_EnableAndIgnoreFutureErrors```

I'd also want to know how many *total* times this all happened, so can we track something this too?

```CodefixInfobar_LeaveDisabled```